src/components/Trade/hooks/useAvailableSwappers.tsx

```diff
@@ -64,18 +64,9 @@ export const useAvailableSwappers = () => {
       // Capture the request start time so we can resolve race conditions when setting swapper/quote data
       const requestStartedTimestamp = Date.now()
 
-      const availableSwapperTypesWithQuoteMetadataResponse =
-        tradeQuoteArgs && feeAsset
-          ? await dispatch(
-              getAvailableSwappers.initiate({
-                ...tradeQuoteArgs,
-                feeAsset,
-                buyAssetFiatRate,
-                sellAssetFiatRate,
-                feeAssetFiatRate,
-              }),
-            )
-          : undefined
+      const availableSwapperTypesWithQuoteMetadataResponse = tradeQuoteArgs
+        ? await dispatch(getAvailableSwappers.initiate(tradeQuoteArgs))
```

Comment:
there is a skip pattern you can use too but somewhat stylistic comment

Reply:
love this idea,  it would be way clear to use the skip token from rtk query:

```
  const { data: availableSwapperTypesWithQuoteMetadataResponse, error, isLoading } = useGetAvailableSwappersQuery(tradeQuoteArgs ?? skipToken)
```

...but this file is riddled with race condition handling logic and refactoring this is better done in a different PR to minimise surface area.

![image](https://github.com/shapeshift/web/assets/125113430/9d94af07-ea53-457a-ae72-8991d4713643)
![image](https://github.com/shapeshift/web/assets/125113430/9ad82958-4756-4ebc-bbd4-963552ced06b)